grommet-leaflet-core/src/Controls.jsx

```diff
@@ -26,10 +26,12 @@ export const LocationBounds = ({ locations }) => {
 
 const Controls = ({ locations }) => {
   const map = useMap();
-  const bounds = LocationBounds({ locations });
+  const bounds = locations ? LocationBounds({ locations }) : undefined;
```

Comment:
Is this the same?

```suggestion
  const bounds = locations && LocationBounds({ locations });
```

Reply:
I think the only difference is && will return a boolean (aka false when no locations) as opposed to undefined. I think since we're later checking `if (bounds)` then resolving to `false` is fine.

Reply:
Done.